src/raven.js

```diff
@@ -5,6 +5,8 @@
 // since JSON is required to encode the payload
 var _Raven = window.Raven,
     hasJSON = !!(typeof JSON === 'object' && JSON.stringify),
+    // Raven can run in contexts where there's no document (react-native)
+    hasDocument = typeof document !== 'undefined',
```

Comment:
On second thought – `isUndefined` is declared halfway through the file, and I like the declaration of `hasDocument` at the very top in that big var statement.